runtime/serde/serde-json/common/test/aws/smithy/kotlin/runtime/serde/json/JsonDeserializerIgnoresKeysTest.kt

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package aws.smithy.kotlin.runtime.serde.json
+
+import aws.smithy.kotlin.runtime.serde.SdkFieldDescriptor
+import aws.smithy.kotlin.runtime.serde.SdkObjectDescriptor
+import aws.smithy.kotlin.runtime.serde.SerialKind
+import aws.smithy.kotlin.runtime.serde.deserializeStruct
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class JsonDeserializerIgnoresKeysTest {
```

Comment:
**Suggestion**: Every test has explicitly modeled field descriptors. You're missing coverage for unmodeled fields that we want to skip rather than enumerate as unknown.